compiler/rustc_span/src/def_id.rs

```diff
@@ -159,6 +159,7 @@ impl DefId {
         DefId { krate: LOCAL_CRATE, index }
     }
 
+    /// Returns whether the item is defined in the currently compiled crate.
```

Comment:
nit: grammar
```suggestion
    /// Returns whether the item is defined in the crate currently being compiled.
```

Reply:
I'm sorry. Accidentally marked this as revolved. Applied the suggestion.